### DISCUSSION TITLE
[Feature] Locational Prompts

### DISCUSSION THREAD

**Discussion by torridgristle** [eps696/aphantasia#16] [Ideas]
What if image slices from the left and image slices from the right were compared to different prompts? Slices in the center and slices at the edge?

Perhaps if it was set to check for a jungle on the edges, and for a toucan in the center, it would create an image of a toucan in a jungle with a broad background of jungle.

While an interface for such a thing may get convoluted quickly, as long as it's kept to simple options like left/out and right/in and midpoint with a toggle for left/right or edge/center or up/down or what have you it shouldn't get too outrageous for a user.

**Comment by eps696** [comment]
there's quite huge room for various processing logic and prompts algebra of such kind, which does not bring any advantage to the method in general. feel free to fork this repo for further development of cosmetic usability on your taste.
